src/lib/components/Strip/Strip.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Strip.css';
+
+const Strip = (props) => {
+  let classArray = [];
+  let style = Object.assign({}, props.style);
+  const {
+    colour, image, bordered, padding, children,
+  } = props;
+
+  if (colour) {
+    classArray = classArray.concat(`p-strip--${props.colour}`);
+  }
+
+  if (image) {
+    classArray = classArray.concat('p-strip--image');
+    style = Object.assign(
+      style, { backgroundImage: `url(${image})` },
```

Comment:
We sometimes have to assign more than background-image here so might be worth taking an object?

Reply:
@anthonydillon I'm not sure I understand? Styles are already accepted as an object prop, and this is just tacking on a background-image to that style object